astro/tests/integration_test.go

```diff
@@ -272,12 +268,20 @@ func TestVersionWithLdflags(t *testing.T) {
 
 func TestPlanErorrsWithoutConfig(t *testing.T) {
 	dir, err := ioutil.TempDir("/tmp", "astro-tests")
-	if err != nil {
-		panic(err)
-	}
+	require.NoError(t, err)
 	defer os.RemoveAll(dir)
 	result := RunTest(t, []string{"plan"}, "/tmp/astro-tests", "")
 	assert.Equal(t, "unable to find config file\n", result.Stderr.String())
 	assert.Equal(t, "", result.Stdout.String())
 	assert.Equal(t, 1, result.ExitCode)
 }
+
+func TestApplyErorrsWithoutConfig(t *testing.T) {
+	dir, err := ioutil.TempDir("/tmp", "astro-tests")
+	require.NoError(t, err)
+	defer os.RemoveAll(dir)
```

Comment:
Same here.

You could also make these two tests into a table test (if you wanted... 😉 )